app/uploaders/logo_uploader.rb

```diff
@@ -29,13 +29,13 @@ def content_type_allowlist
 
   def filename
     # random_string in the filename to avoid caching issues
-    "original_logo_#{random_string}.#{file.extension}" if original_filename
```

Comment:
This is a bit of an unrelated change but part of the same feature, we're not using the `original_filename` as part of the new file name, hence there is no need to check for it.